code/frameworks/experimental-nextjs-vite/package.json

```diff
@@ -93,21 +98,20 @@
     "@storybook/builder-vite": "workspace:*",
     "@storybook/react": "workspace:*",
     "@storybook/test": "workspace:*",
-    "styled-jsx": "5.1.6"
+    "styled-jsx": "5.1.6",
+    "vite-plugin-storybook-nextjs": "^1.0.9"
   },
   "devDependencies": {
     "@types/node": "^18.0.0",
     "next": "^14.2.5",
-    "typescript": "^5.3.2",
-    "vite-plugin-storybook-nextjs": "^1.0.0"
```

Comment:
Any reason not to prebundle?

Reply:
The plugin has a specific package.json export map, which is important. Also, prebundling avoids the user's possibility to install a specific version and locks us in into the semver release cycle of Storybook.